commands/user_test.go

```diff
@@ -52,3 +52,302 @@ func (s *MmctlUnitTestSuite) TestSearchUserCmd() {
 		s.Require().Equal("Unable to find user 'example@example.com'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestUserInviteCmd() {
+	s.Run("Invite user to an existing team by Id", func() {
+		printer.Clean()
+		argUser := "example@example.com"
+		argTeam := "teamId"
+
+		s.client.
+			EXPECT().
+			GetTeam(argTeam, "").
+			Return(&model.Team{Id: "teamId"}, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			InviteUsersToTeam(argTeam, []string{argUser}).
+			Return(false, &model.Response{Error: nil}).
+			Times(1)
+
+		err := userInviteCmdF(s.client, &cobra.Command{}, []string{argUser, argTeam})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Equal("Invites may or may not have been sent.", printer.GetLines()[0])
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
+
+	s.Run("Invite user to an existing team by name", func() {
+		printer.Clean()
+		argUser := "example@example.com"
+		argTeam := "teamName"
+		resultID := "teamId"
+
+		s.client.
+			EXPECT().
+			GetTeam(argTeam, "").
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetTeamByName(argTeam, "").
+			Return(&model.Team{Id: resultID}, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			InviteUsersToTeam(resultID, []string{argUser}).
+			Return(false, &model.Response{Error: nil}).
+			Times(1)
+
+		err := userInviteCmdF(s.client, &cobra.Command{}, []string{argUser, argTeam})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Equal("Invites may or may not have been sent.", printer.GetLines()[0])
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
+
+	s.Run("Invite user to several existing teams by name and id", func() {
+		printer.Clean()
+		argUser := "example@example.com"
+		argTeam := []string{"teamName1", "teamId2", "teamId3", "teamName4"}
+		resultTeamModels := [4]*model.Team{
+			&model.Team{Id: "teamId1"},
+			&model.Team{Id: "teamId2"},
+			&model.Team{Id: "teamId3"},
+			&model.Team{Id: "teamId4"},
+		}
+
+		// Setup GetTeam
+		s.client.
+			EXPECT().
+			GetTeam(argTeam[0], "").
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetTeam(argTeam[1], "").
+			Return(resultTeamModels[1], &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetTeam(argTeam[2], "").
+			Return(resultTeamModels[2], &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetTeam(argTeam[3], "").
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		// Setup GetTeamByName
+		s.client.
+			EXPECT().
+			GetTeamByName(argTeam[0], "").
+			Return(resultTeamModels[0], &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetTeamByName(argTeam[1], "").
+			Times(0)
+
+		s.client.
+			EXPECT().
+			GetTeamByName(argTeam[2], "").
+			Times(0)
```

Comment:
These two expectations are safe to remove

Reply:
Not sure how I forgot these ones on the previous sweep. Thanks for pointing them out!